docs/builtin-plugins.md

```diff
@@ -4,22 +4,72 @@ Fusebox contains premade plugins, that should help you to get started.
 
 ## CSS Plugin
 
-It's very easy to start working with css files. You have 2 options, you either bundle the contents or serve the files. A decision that can be made at build time.
+The CSS plugin is elegant and powerful.  It can be used to combine all CSS files into a single bundle, or to mark files as external to
+the bundle, to be served separately by the web server.  By default, all CSS files are bundled.
 
 For example:
 ```js
 plugins: [
+    fsbx.CSSPlugin({})
+]
+```
+
+### Chaining CSS plugins to transform SCSS, Sass, and others
+
+Fusebox's plugin system allows simple transformation of more complex CSS systems through plugin chaining.  For example, to transform
+Sass into CSS:
+
+```
```

Comment:
marked for this and all the other code blocks (most had no marking, from the original doc)